README.md

```diff
@@ -151,6 +151,8 @@ async function subsystem (fastify, opts) {
 
 To use this module with TypeScript, make sure to install `@types/express`.
 
+In addition, you will need to add `"types": ["fastify-express"]` to your tsconfig.json file when using `require` to import the plugin, otherwise it will work out of the box.
```

Comment:
Wouldn't it be "otherwise it will not work out of the box"?

Reply:
if (require) "add types"
else "will work"

I think is well written, do you prefer to say it with different words?

Reply:
See other response, just needed cutting down.